### ISSUE TITLE
Bring JupyterLite into this repo

### ISSUE THREAD

**Issue by pauleveritt** [pauleveritt/tagstr-site#2]
I think Koudai's JupyterLite playground is going to be the number one way that people poke around with this. It's compelling.

Koudai, if you don't mind, can we set it up in this repo? I can then link to it from the README and work with you on some examples to go into the notebook's text.

We might have multiple notebooks (or pages in a notebook) for different uses: a very simple example, HTML templating, etc.

**Comment by koxudaxi** [comment]
@pauleveritt 
OK, I will bring the playground here this weekend.
Can I use precisely the same method of deployment, etc.?

**Comment by pauleveritt** [comment]
@koxudaxi Yes, precisely the same. I think that was great.

Do you think we should have multiple notebooks for different topics, or just different pages per topic?

**Comment by koxudaxi** [comment]
It seems like it would be a good idea to have a note for each scenario such as html, SQL, etc.?
I am not that familiar with notes, but it would be faster to actually try it.

**Comment by pauleveritt** [comment]
Any chance you have time this week to work on it? I could provide the text and examples for some pages.

**Comment by koxudaxi** [comment]
I had a busy weekend and went to bed early at night. This is the priority for this week's OSS task.
I will now port JupyterLite from my repo.

**Comment by pauleveritt** [comment]
No rush at all. I'm sure it's been quite a week for you. Really enjoyed spending time together at PyCon.

**Comment by koxudaxi** [comment]
> No rush at all. I'm sure it's been quite a week for you. Really enjoyed spending time 

I had no idea it would be such a fun event! 😄 
But it was a bit tired, unlike the days when I worked on the computer at home every day 😅

**Comment by koxudaxi** [comment]
@pauleveritt 
I tested a new notebook in my playground repo.
We can add the notebooks to `content` directory to show up in the site.
If you convert the PEP or tutorial rst into a notebook as it is, it may be difficult to understand.
Also I am not familiar with notebooks, I just added a new notebook in PyCharm.
https://github.com/koxudaxi/tagstr-playground/pull/3

https://koxudaxi.github.io/tagstr-playground/lab/index.html

<img width="1277" alt="image" src="https://github.com/pauleveritt/tagstr-site/assets/630670/87a7998d-d7b8-419c-b659-9e5730c56317">

**Comment by pauleveritt** [comment]
I agree that we shouldn't convert the PEP or tutorial as it is now. We should make something that is a good fit for a notebook.
